packages/library-sync/src/androidMain/AndroidManifest.xml

```diff
@@ -15,4 +15,9 @@
   ~ limitations under the License.
   -->
 
-<manifest package="io.realm.mongodb.sync" />
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="io.realm.mongodb.sync">
```

Comment:
Maybe just `io.realm.mongodb` as this contains more than just sync.

We should probably figure out if `library-sync` is the best name for this module/package since it contains a lot more than just sync, but neither `library-app` nor `library-mongodb` seems that much better.

Something to think about before we announce a public release.